<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AritmatikaOperation</class>
 <widget class="QMainWindow" name="AritmatikaOperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1336</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>391</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>391</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>70</y>
      <width>391</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gambar 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gambar 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hasil</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1336</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGambar_1"/>
    <addaction name="actionGambar_2"/>
    <addaction name="actionSimpan"/>
   </widget>
   <widget class="QMenu" name="menuPertitungan">
    <property name="title">
     <string>Pertitungan</string>
    </property>
    <addaction name="actionPenjumlahan"/>
    <addaction name="actionPengurangan"/>
    <addaction name="actionPembagian"/>
    <addaction name="actionPerkalian"/>
   </widget>
   <widget class="QMenu" name="menuNotasi">
    <property name="title">
     <string>Notasi</string>
    </property>
    <addaction name="actionOR"/>
    <addaction name="actionAND"/>
    <addaction name="actionNOT"/>
    <addaction name="actionXOR"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPertitungan"/>
   <addaction name="menuNotasi"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGambar_1">
   <property name="text">
    <string>Gambar 1</string>
   </property>
  </action>
  <action name="actionGambar_2">
   <property name="text">
    <string>Gambar 2</string>
   </property>
  </action>
  <action name="actionSimpan">
   <property name="text">
    <string>Simpan</string>
   </property>
  </action>
  <action name="actionPenjumlahan">
   <property name="text">
    <string>Penjumlahan</string>
   </property>
  </action>
  <action name="actionPengurangan">
   <property name="text">
    <string>Pengurangan</string>
   </property>
  </action>
  <action name="actionPembagian">
   <property name="text">
    <string>Pembagian</string>
   </property>
  </action>
  <action name="actionPerkalian">
   <property name="text">
    <string>Perkalian</string>
   </property>
  </action>
  <action name="actionOR">
   <property name="text">
    <string>OR</string>
   </property>
  </action>
  <action name="actionAND">
   <property name="text">
    <string>AND</string>
   </property>
  </action>
  <action name="actionNOT">
   <property name="text">
    <string>NOT</string>
   </property>
  </action>
  <action name="actionXOR">
   <property name="text">
    <string>XOR</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
